src/components/StatusBar.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC } from "react";
+import classnames from "classnames";
+import Version from "./Version";
+import OperationStatus from "./OperationStatus";
+import { useToastNotification } from "context/toastNotificationProvider";
+import { ICONS, Icon } from "@canonical/react-components";
+import { iconLookup, severityOrder } from "util/notifications";
+
+interface Props {
+  className?: string;
+}
+
+const StatusBar: FC<Props> = ({ className }) => {
+  const { toggleListView, notifications, countBySeverity, isListView } =
+    useToastNotification();
+
+  const notificationIcons = severityOrder.map((severity) => {
+    if (countBySeverity[severity]) {
+      return (
+        <Icon
+          key={`status-${severity}`}
```

Comment:
```suggestion
          key={severity}
```
Keys can be as simple as possible